using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -446,24 +510,30 @@ Identity.getUrlVariables(new AdobeCallback<String>() {
 {% tab title="iOS" %}
 ### getUrlVariables <a id="geturlvariables-ios"></a>
 
-_added in ACPCore v2.3.0 \(ACPIdentity v2.1.0\)_
+{% hint style="info" %}
+This method was added in ACPCore version 2.3.0 and ACPIdentity version 2.1.0.
+{% endhint %}
 
-Retrieve Adobe visitor data as a URL query parameter string for consumption in hybrid mobile applications. There is no leading "?" or "&" punctuation, as the caller is responsible for placing the string in the correct location of their resulting URL. The following information is added to the string that is returned via the callback:
+* This `getUrlVariables()` API gets the Visitor ID Service variables in URL query parameter form for consumption in hybrid app. This method will return an appropriately formed string containing Visitor ID Service URL variables. There will be no leading (&) or (?) punctuation, as the caller is responsible for placing it in their resulting java.net.URI in the correct location.
 
-* The adobe\_mc attribute is an URL encoded list that contains:
-  * `MCMID` - Experience Cloud ID \(ECID\)
-  * `MCORGID` - Experience Cloud Org ID
-  * `MCAID` - Analytics Tracking ID \(AID\), if available from the [Analytics extension](../../adobe-analytics/)
-  * `TS` - A timestamp taken when this request was made
-* The optional `adobe_aa_vid` attribute is the URL-encoded Analytics Custom Visitor ID \(VID\), if previously set in the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/174e9069bc1d3a521b59d52a066e9a7730f60ff5/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#setidentifier).
+  If an error occurs while retrieving the URL string, (*callback*) will be called with a null value. Otherwise, the following information is added to the string that is returned in the callback as an [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback) instance:
+
+  - The `adobe_mc` attribute is an URL encoded list that contains:
+    - `MCMID` - Experience Cloud ID \(ECID\)
+    - `MCORGID` - Experience Cloud Org ID
+    - `MCAID` - Analytics Tracking ID \(AID\), if available from the [Analytics extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics)
+    - `TS` - A timestamp taken when this request was made
+  - The optional `adobe_aa_vid` attribute is the URL-encoded Analytics Custom Visitor ID \(VID\), if previously set in the [Analytics extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics).
 
 #### **Syntax**
 
 ```objectivec
 + (void) getUrlVariables: (nonnull void (^) (NSString* __nullable urlVariables)) callback;
 ```
 
-#### **Examples**
+o   *callback* is a block pointer to call with an NSString value that contains the visitor identifiers as a querystring upon completion of the service request.
```

Comment:
- Please remove "o" and keep the asterisk for the bullet point.
- Please reword to read:
"*callback* has an NSString value that contains the visitor identifiers as a querystring after the service request is complete.